nixos/modules/services/web-apps/nextcloud.nix

```
@@ -1002,6 +1020,19 @@ in {
               exit 1
             fi
 
+            ${optionalString (!cfg.disablePathSafetyCheck) ''
+              # Check if systemd-tmpfiles setup worked correctly
+              for i in config data; do
```

```suggestion
              for i in config/config.php data; do
```
Nextcloud only checks the owner of the config.php file: https://github.com/nextcloud/server/blob/fdc0b1ecf423e400257edf868a51d5873fc66c64/console.php#L49